Makefile

```diff
@@ -33,8 +33,6 @@ ruby:
 	openapi-generator-cli generate -i tmp/compiled.yaml -g ruby -o clients/ruby -c ./openapi-generator/ruby_lang.yaml
 go:
 	openapi-generator-cli generate -i tmp/compiled.yaml -g go -o clients/go -c ./openapi-generator/go_lang.yaml
-	go install golang.org/x/tools/cmd/goimports@latest
-	goimports -w clients/go
```

Comment:
Was getting the [following](https://github.com/phrase/openapi/actions/runs/6586658228/job/17895395665#step:3:578) issue with these two lines, looked for a fix which suggested updating system's bashprofile. Tested with remove them, apparently it works, looked up briefly what [goimport](https://pkg.go.dev/golang.org/x/tools/cmd/goimports) does, apparently it is some tidying up around imports.

Reply:
Might be needed after all, fixing it.